src/wp-content/themes/twentysixteen/js/functions.js

```diff
@@ -4,210 +4,290 @@
  *
  * Contains handlers for navigation and widget area.
  */
+ ( function () {
+	'use strict';
 
-( function( $ ) {
-	var body, masthead, menuToggle, siteNavigation, socialNavigation, siteHeaderMenu, resizeTimer;
+	var masthead, menuToggle, siteNavigation, socialNavigation, siteHeaderMenu, resizeTimer;
+
+	function matches( el, sel ) {
```

Comment:
As noted in the trac ticket, this and the following two functions are mostly intended to preserve IE 11 compatibility. If that's not desirable, we can remove them and make the code smaller.

Reply:
I think it would be best to discuss this on the Trac tickets (also affects the similar PRs #1682 and #1683). My take on this is that we should not include this extra code for IE11 here, since support has officially been dropped. What we could do is use the `wp_get_script_polyfill()` approach you've suggested in https://core.trac.wordpress.org/ticket/55126#comment:3, e.g. we could do that only if that function exists. In that case we would at least provide minimal support - although given the official stance I'm not even sure that is needed.

Reply:
Agreed, let's continue the discussion on Trac. I have no strong opinions on the matter, and am happy to change the code to fit whatever we decide on.